srcalc/src/main/java/gov/va/med/srcalc/security/VistaLinkVistaDaoFactory.java

```diff
@@ -0,0 +1,39 @@
+package gov.va.med.srcalc.security;
+
+import org.springframework.security.core.Authentication;
+import org.springframework.security.core.context.SecurityContextHolder;
+
+import gov.va.med.srcalc.vista.*;
+import gov.va.med.srcalc.vista.vistalink.VistaLinkProcedureCaller;
+
+public class VistaLinkVistaDaoFactory implements VistaDaoFactory
```

Comment:
Perhaps, but I feel too much time pressure to change it now. We can change it later if we care.